### ISSUE TITLE
FltQuery.run() doesn't work when 'Flight Date (exact)' is selected - tz TypeError

### ISSUE THREAD

**Issue by mattsq** [ge-flight-analytics/emspy#59]
Started giving emspy a go through the new RStudio reticulate::python_repl() - it works pretty well! I did notice one thing that was weird, which I can replicate in Jupyter, so it's not the RStudio environment specifically - any query that includes 'Flight Date (exact)' fails out with a ```TypeError: Already tz-aware, use tz_convert to convert.```.

Here's a minimal script that causes the issue - think it's probably related to this issue:

[https://github.com/influxdata/influxdb-python/issues/671](url)

```
import emspy
import emspy.query as query

c = emspy.Connection(user="***.***", pwd="****", server_url="https://qfa-api.au.efoqa.com/api")

qry = query.FltQuery(conn=c, ems_name="QANTAS-EMS", data_file="test.db")

qry.set_database("FDW Flights")
qry.generate_preset_fieldtree()
qry.select("flight record", "takeoff airport code", "flight date")
qry.filter("'tail number' == 'VH-ZNA'")
qry.filter("'flight date' >= '2020-1-1'")
qry.get_top(n = 200)

q = qry.run()
```
And here's the traceback that Jupyter dumped:
```
<ipython-input-1-6a86fc671913> in <module>
     13 qry.get_top(n = 200)
     14 
---> 15 q = qry.run()

~/Library/r-miniconda/envs/emspytest/lib/python3.8/site-packages/emspy/query/fltquery.py in run(self, n_row)
    512 
    513         if (Nout is not None) and (Nout <= 25000):
--> 514             return self.simple_run(output="dataframe")
    515 
    516         return self.async_run(n_row=n_row)

~/Library/r-miniconda/envs/emspytest/lib/python3.8/site-packages/emspy/query/fltquery.py in simple_run(self, output)
    421             return content
    422         elif output == "dataframe":
--> 423             return self.__to_dataframe(content)
    424         else:
    425             raise ValueError("Requested an unknown output type.")

~/Library/r-miniconda/envs/emspytest/lib/python3.8/site-packages/emspy/query/fltquery.py in __to_dataframe(self, json_output)
    558                     df.iloc[:, i] = df.iloc[:, i].astype(bool)
    559                 elif ctype == 'dateTime':
--> 560                     df.iloc[:, i] = pd.to_datetime(df.iloc[:, i]).dt.tz_localize('UTC')
    561             except ValueError:
    562                 print("Somethings wrong when converting to Pandas DataFrame for column '%s' "

~/Library/r-miniconda/envs/emspytest/lib/python3.8/site-packages/pandas/core/accessor.py in f(self, *args, **kwargs)
     97         def _create_delegator_method(name):
     98             def f(self, *args, **kwargs):
---> 99                 return self._delegate_method(name, *args, **kwargs)
    100 
    101             f.__name__ = name

~/Library/r-miniconda/envs/emspytest/lib/python3.8/site-packages/pandas/core/indexes/accessors.py in _delegate_method(self, name, *args, **kwargs)
    102 
    103         method = getattr(values, name)
--> 104         result = method(*args, **kwargs)
    105 
    106         if not is_list_like(result):

~/Library/r-miniconda/envs/emspytest/lib/python3.8/site-packages/pandas/core/indexes/datetimes.py in tz_localize(self, tz, ambiguous, nonexistent)
    233         self, tz, ambiguous="raise", nonexistent="raise"
    234     ) -> "DatetimeIndex":
--> 235         arr = self._data.tz_localize(tz, ambiguous, nonexistent)
    236         return type(self)._simple_new(arr, name=self.name)
    237 

~/Library/r-miniconda/envs/emspytest/lib/python3.8/site-packages/pandas/core/arrays/datetimes.py in tz_localize(self, tz, ambiguous, nonexistent)
    963                 new_dates = tzconversion.tz_convert_from_utc(self.asi8, self.tz)
    964             else:
--> 965                 raise TypeError("Already tz-aware, use tz_convert to convert.")
    966         else:
    967             tz = timezones.maybe_get_tz(tz)
```

**Comment by danverkamp** [comment]
@mattsq 

Hey Matt, thanks for pointing this out and providing a good code snippet! I certainly wasn't aware of the issue. And sorry for the delay here. I didn't get a notification until you pinged me yesterday. 

Let me take a look and see what's going on. 

Thanks, 

Daniel

**Comment by mattsq** [comment]
All good! We replicated the issue with some other people in the team - some people's environments seemed to be OK, but we didn't get all the way down to what actually fixes it. My PR fixes the issue, but I'm not sure it's actually the right thing to do - I'm not sure what tz the 'Flight Date (Exact)' field is returning prior to being passed to:

`df.iloc[:, i] = pd.to_datetime(df.iloc[:, i]).dt.tz_localize('UTC')`

**Comment by danverkamp** [comment]
@mattsq 

I just saw the Pull Request too! I'm slacking. I'll take a look. I wonder if it's because you're using 3.8

**Comment by mattsq** [comment]
This is fixed, yay! Via #60
